azure-pipelines.yml

```diff
@@ -38,8 +38,8 @@ variables:
     value: "Any CPU"
   - name: SOLUTION
     value: "SonarScanner.MSBuild.sln"
-  - name: NUGET_VERSION
-    value: "6.10.1"
+  - name: NUGETEXETOOLPATH
```

Comment:
Once we get rid of the NuGetCommand constraint, this could go away, as nuget will be on path?